### ISSUE TITLE
Drop the phpunit dependency

### ISSUE THREAD

**Issue by mdeboer** [martin-helmich/phpunit-json-assert#15]
With Symfony 4 there is no hardcoded phpunit dependency when using the phpunit bridge. Installing this package automatically installs phpunit as none is detected.

Using `provide` in the composer.json file doesn't work either to fake the phpunit installation.

**Comment by martin-helmich** [comment]
This package depends on specific versions of PHPUnit (see the [compatibility section in the `README.md`](https://github.com/martin-helmich/phpunit-json-assert#compatibility)). Removing the PHPUnit dependency at this point would completely break version resolution (for example, without an explicit PHPUnit dependency, users of PHPUnit 5 would have this package upgrade to version 2.* on `composer update`, which actually requires PHPUnit >= 6 to work).

To sum up, that's a regretful "no" on this issue. I'm open to backwards-compatible suggestions, however.

**Comment by eXtreme** [comment]
@martin-helmich shouldn't [`conflict`](https://getcomposer.org/doc/04-schema.md#conflict) be used for such constraint? Define that 1.x branch conflicts with `phpunit/phpunit` `>=6.0` and 2.x conflicts with `<6.0`.

**Comment by martin-helmich** [comment]
@eXtreme That might just work; thanks for the suggestion. :+1: I'm going to fool around a little bit with that as soon as I find the time.

**Comment by tonivdv** [comment]
Hi @martin-helmich ,

You library is really useful. However when used with Symfony solving this issue would highly be appreciated. The referenced fix by @mlazze could do the trick?! 

De you still plan to maintain your library?

Cheers

**Comment by martin-helmich** [comment]
Fixed in #16. Sorry for the delay. Using `conflict` as suggested by @eXtreme seems to work just fine.
